src/i18n/locales/hi/translation.json

```diff
@@ -4,7 +4,7 @@
       "nav": {
         "welcome": "नमस्कार",
         "password": "आपका पासवर्ड",
-        "connect": "आपका लाइटनिंग खाता",
+        "connect": "आपका Lightning खाता",
```

Comment:
Hey @legendarykamal , I've done that intentionally.

The word "lightning" directly converted to hindi does not have any specific meaning.
It can be confused with something that's realated to electricity by new users.
I wanted to make sure that new users understanding "Lightning ⚡" as a protocol and not with it's actual meaning.

Hope that helps.

Reply:
Okay, I'll do it if you insist. 👍🏻